### DISCUSSION TITLE
Dark Ages: Mage character sheet?

### DISCUSSION THREAD

**Discussion by Fenlander216** [JohanFalt/Foundry_WoD20#874] [Q&A] [answered]
Hello. I like the look of the system you have made for Dark Ages Vampire. I intend to run one in the future for my players. Would it also be possible to create a character sheet for Dark Ages: Mage as well?

**Comment by JohanFalt** [comment, marked as answer]
I looked at this a while back and came to the conclusion that most likely not.

**Reply by Fenlander216** [reply]
Okay, thank you for replying.
